cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/AbstractParameters.java

```diff
@@ -31,12 +32,17 @@ public AbstractParameters(HttpServletRequest request) {
   }
 
   protected void initParameters() throws UnsupportedEncodingException {
+    _initialized = true;
     _endPoint = ParameterUtils.endPoint(_request);
     _json = ParameterUtils.wantJSON(_request);
   }
 
   @Override
   public boolean parseParameters(HttpServletResponse response) {
+    if (_initialized) {
+      LOG.debug("Attempt to parse the request {} which has already been parsed.", _request);
```

Comment:
Nit: Should we make it `trace` level? Also would this change make the message more concise? -> `Attempt to parse an already parsed request {}.`